build.gradle

```diff
@@ -5,10 +5,14 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.3'
+        classpath 'com.android.tools.build:gradle:8.11.1'
     }
 }
 
+plugins {
+    id 'org.jetbrains.kotlin.android' version '1.8.0' apply false
+}
```

Comment:
Add these lines to suppress ` Duplicate class kotlin.collections.jdk8.CollectionsJDK8Kt found` error.